x-pack/plugin/eql/qa/rest/src/test/resources/rest-api-spec/test/eql/10_basic.yml

```diff
@@ -9,6 +9,7 @@ setup:
               _id:    1
           - event:
               - category: process
+              - sequence: 1
```

Comment:
Currently the tie breaker is mandatory - which makes sense for ECS and our tests but not the rest, I plan to make it optional.